### DISCUSSION TITLE
🎉New Search Engine Feature Suggestions 🔥

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5763] [Working on Features]
[🎉New Search Engine Feature Suggestions 🔥](https://community.fluxbuilder.com/c/suggestions-feedback/new-search-engine-feature-suggestions)

hello, as you know, the integrated ajax search plugin does not work very well. There are many known problems, for example, product prices and galleries are not visible, or more than 10 products are not visible.

I have been using fluxstore for 2 years and I am much happier with the updates made day after day. I wish you continued success, I hope you will come across with much more beautiful products, please pay attention to the features that I will mention a little later, this is very important for us and our users, as you know, if our customers cannot find the products they want to buy, we cannot sell.

-Suggestions-
1.fluxbuilder ekranında fluxstore search için düzenleme ekranı eklenebilir. kategori ismine göre sonuç ver  ,etiket dahil ara  , attribute dahil ara , Custom Taxonomies dahil sonuç ver veya ACF

Example : Our customer did a search for "Grill" and the title of the product the customer requested reads "Blaze Professional LUX 3-Burner Built-In Gas" and the product is added in the "Grill" category. If this feature is added, the product will match again from the category section.

Example: Our customer made a search for "Apple" and it says "iphone 14 pro" from the title of the product the customer wants. If this feature is added, the product will match again from the brand section.

2.Fuzzy search 

https://fibosearch.com/documentation/features/fuzzy-search/

3. By adding the Synonyms feature, it becomes easier for the customer to reach the product they want. this can be as follows: A field can be added to the mstore api plugin and integrated into the application with json output.
![forest_search](https://github.com/inspireui/support/assets/57155882/c198f218-5f2b-4cc2-a861-feb582861abf)
![synonyms1](https://github.com/inspireui/support/assets/57155882/543899fd-42b5-44b5-8aa0-0d0d5037eae5)
![forest_search2](https://github.com/inspireui/support/assets/57155882/3af8b1ea-0a06-433f-8db3-13fdf0af1089)

**Comment by osman-kocak** [comment]
@minhcasi @HanielQC

**Comment by minhcasi** [comment]
@osman-kocak - Our team is currently conducting research on a new built-in search engine for the app builder and FluxStore apps. We anticipate that the team will be prepared in the near future.

**Reply by osman-kocak** [reply]
I found new method, Since it will take a lot of time and effort to provide appropriate support for each search engine, please specify an endpoint url for us, such as example.com/flutter-ex-search/search/results/

After than Show us an example result schema in xml format We will get and post result the link you want from our plugins.

**Comment by merorose** [comment]
listing:
now app can search only on title and description
but app suppose to search in keywords and all content also (as already happen in website)
how come can't search in keywords??

**Reply by osman-kocak** [reply]
obviously a long wait. But the search section currently used is a disgrace for e-commerce.

**Comment by osman-kocak** [comment]
Up
